types/src/raw/collection.rs

```diff
@@ -7,12 +7,16 @@ use uuid::Uuid;
 pub struct EditedCollection {
     /// UUID for the collection
     pub title: String,
+    /// Description of the collection (optional)
+    pub description: Option<String>,
```

Comment:
As we review with the team, we should ask whether this should be optional from a workflow perspective.

Some questions we will have to answer:
1. Does not having a description for a collection even make sense?
2. Is getting a collection structure up quickly more important than adding a description upfront?